docs/cmd/kn_service_create.md

```diff
@@ -54,7 +54,7 @@ kn service create NAME --image IMAGE [flags]
   -p, --port int32               The port where application listens on.
       --requests-cpu string      The requested CPU (e.g., 250m).
       --requests-memory string   The requested memory (e.g., 64Mi).
-      --wait-timeout int         Seconds to wait before giving up on waiting for service to be ready (default: 60). (default 60)
+      --wait-timeout int         Seconds to wait before giving up on waiting for service to be ready. (default 60)
```

Comment:
[nit] maybe move the `.` at the end. So ` service to be ready (default 60).` for consistency with other descriptions.

Reply:
Unfortunately, that's not so easy as the `(default ...)` part is added by cobra without `.`. Tbh, the easiest would be to adhere to cobra's standard, i.e. start with lower case (see `--help`) and no dot at the end.

Reply:
I could remove the cobra default value and just make the default handling on our own (if not set, then use 60). Then I have full control of this help message. Not sure whether its worth the effort though.